andela/eventx
# #101390216 Request Refund

#### What does this PR do?
Users who have paid for tickets to an event that is cancelled can request refunds
#### Description of Task to be completed?
Add feature test for request refunds
Add and run migration for refunds
Include route to request refund
Modify test to accomodate changes
Hide attend button for cancelled events
Create and modify view
Resolve permission conflicts
Use decorators to clean up view
Allow only event managers to manage their events
Use `cancancan` for access control
#### How should this be manually tested?
Become an event manager and create an event with a paid ticket
Add a user account to attend the event by buying a ticket
Log in as an Event manager and cancel the event by clicking disable
Log in as a User, you should see a button to request a refund
#### Also included
#119349579 Event Managers can only manage their own events (bg)
#119542077 Users cannot attend cancelled Events (bg)
#### Screenshots
![image](https://cloud.githubusercontent.com/assets/17007838/15254271/de775d32-192d-11e6-9462-bc97dd808160.png)
![image](https://cloud.githubusercontent.com/assets/17007838/15254293/f60e6ca6-192d-11e6-8b0f-3f1cdf1d7cda.png)
